CardManagementSDK/Source/Views/ViewPin/NIViewPinView.swift

```diff
@@ -0,0 +1,233 @@
+            counter -= 1
+        } else {
+            timer?.invalidate()
+            maskPIN()
+        }
+    }
+    
+    private func maskPIN() {
+        firstDigit.text = "＊"
+        secondDigit.text = "＊"
+        thirdDigit.text = "＊"
+        fourthDigit.text = "＊"
+        fifthDigit.text = "＊"
+        sixthDigit.text = "＊"
+        countDownDecription.isHidden = true
+    }
+    
+    private func updateUI() {
+        guard let viewModel = viewModel else { return }
+        
+        UIFont.registerDefaultFonts()
+        stackView.semanticContentAttribute = .forceLeftToRight
+        
+        viewModel.bindCardDetailsViewModel = {
+            DispatchQueue.main.async {
+                self.activityIndicator.stopAnimating()
+                self.hideUI(false)
+                
+                if viewModel.pin?.count == 4 {
+                    self.digits = [self.firstDigit, self.secondDigit, self.thirdDigit, self.fourthDigit]
+                    self.fourthSeparator.isHidden = true
+                    self.fifthDigit.isHidden = true
+                    self.fifthSeparator.isHidden = true
+                    self.sixthDigit.isHidden = true
+                    self.updateFirst4Digits(viewModel: viewModel)
+                } else if viewModel.pin?.count == 5 {
+                    self.digits = [self.firstDigit, self.secondDigit, self.thirdDigit, self.fourthDigit, self.fifthDigit]
+                    self.fourthSeparator.isHidden = false
+                    self.fifthDigit.isHidden = false
+                    self.fifthSeparator.isHidden = true
+                    self.sixthDigit.isHidden = true
+                    self.updateFirst4Digits(viewModel: viewModel)
+                    self.fifthDigit.text = viewModel.pin?[4]
+                } else if viewModel.pin?.count == 6 {
+                    self.digits = [self.firstDigit, self.secondDigit, self.thirdDigit, self.fourthDigit, self.fifthDigit, self.sixthDigit]
+                    self.fourthSeparator.isHidden = false
+                    self.fifthDigit.isHidden = false
+                    self.fifthSeparator.isHidden = false
+                    self.sixthDigit.isHidden = false
+                    self.updateFirst4Digits(viewModel: viewModel)
+                    self.fifthDigit.text = viewModel.pin?[4]
+                    self.sixthDigit.text = viewModel.pin?[5]
+                }
+                
+                if viewModel.startTimer && self.counter != 0 {
+                    self.startTimer()
+                    let countDownDescriptionLocalized: String = "view_pin_countdown_description".localized + " "
+                    let countDownUnitLocalized: String = " " + "view_pin_countdown_unit".localized
+                    self.countDownDecription.text = countDownDescriptionLocalized + String(Int(self.counter)) + countDownUnitLocalized
```

Comment:
Clarify that same concatenation is applicable for RTL.

Reply:
Looks fine in tests.